packages/scene-composer/src/components/panels/scene-settings/MatterportIntegration.tsx

```diff
@@ -85,23 +89,28 @@ export const MatterportIntegration: React.FC = () => {
     return null;
   }, [connectionOptions, selectedConnectionName]);
 
-  const onMatterportModelIdChange = useCallback(
-    (event) => {
-      const value = event.detail.value;
-      if (value !== matterportModelId) {
-        setMatterportModelId(value);
-        setDirty(true);
-      }
-    },
-    [setMatterportModelId, setDirty],
-  );
+  const onInputBlur = useCallback(() => {
+    if (getSceneProperty(KnownSceneProperty.MatterportModelId) !== matterportModelId) {
+      setSceneProperty(KnownSceneProperty.MatterportModelId, matterportModelId !== '' ? matterportModelId : undefined);
+    }
+    setFocusInput(false);
+  }, [matterportModelId, setFocusInput]);
+
+  const onMatterportModelIdChange = useCallback((event) => {
+    const value = event.detail.value;
+    if (value !== matterportModelId) {
+      setMatterportModelId(value);
+    }
+  }, []);
```

Comment:
you will need to add matterportModelId as dependency, otherwise it will always compares with the original value

Reply:
added matterportModelId as dependency